setup.sh

```diff
@@ -8,6 +8,21 @@ echo Installing posgresql
 sudo apt install -y postgresql postgresql-contrib libpq-dev
 sudo apt install -y python3-pip
 
+echo Installing redis-server
```

Comment:
Redis, I see. Are you going to use this instead of RabbitMQ? I've never used Redis, mostly because when I have dealt with queuing (Celery) RabbitMQ seemed like the more straightforward option, and Redis seems to have some proprietary bits (if you want ssl for an internet-facing connection apparently you have to subscribe to an "enterprise version").

Nevertheless I hear that Redis has awesome performance, which makes sense as essentially it's an in-memory database with some communication features. Looking forward to seeing where this goes.